debian/rules

```diff
@@ -1,33 +1,70 @@
 #!/usr/bin/make -f
+#include /usr/share/dpkg/default.mk
+#include /usr/share/dh-php/pkg-pecl.mk
 
 DH_VERBOSE = 1
 
-DPKG_EXPORT_BUILDFLAGS = 1
-include /usr/share/dpkg/default.mk
+# already in pkg-pecl.mk
+#DPKG_EXPORT_BUILDFLAGS = 1
 
-export DEB_BUILD_MAINT_OPTIONS = hardening=+all
```

Comment:
I would like to keep this here, even if it's already defined somewhere else: we want to be __sure__ that we're compiling with hardening flags.

Reply:
pkg-pecl.mk does it for you

$ grep harde /usr/share/dh-php/pkg-pecl.mk 
export DEB_BUILD_MAINT_OPTIONS = hardening=+all

But it makes sense to be sure it is defined

Reply:
[Not all of them are doing it]( https://github.com/openwrt/packages/blob/master/lang/php7/pecl.mk ) actually.